### PR TITLE
Update PSA services documentation

### PR DESCRIPTION
**Pull request by medou-boushab**

### CODE REVIEW

File: src/docs/PSA/references/Service_Execution.adoc [hunk 1]
```diff
@@ -2,5 +2,5 @@ PSA services can be instantiated in two different ways:
 
 1- Via the <<_psa_portal,ProActive Service Automation>> (PSA) Portal.
 
-2- Via standard workflows (also called *Management Workflows*). ProActive includes predefined standard workflows that manage PSA service lifecycles (e.g., launch and stop PSA services).
-These workflows are provided in the bucket *Service_Automation* under the project *Service Automation - Management*. Their names are constructed using the pattern *PSA_Service_Name* followed by the suffix *_Service*.
\ No newline at end of file
+2- Via Workflow Execution Portal. To launch PSA service, you have to open the Submit Template and choose PSA service from the selected bucket. At this point, you can provide values for variables that help configure this service then press submit.
```
Comment (mboussaa) [inline src/docs/PSA/references/Service_Execution.adoc:5]: 2- Via Workflow Execution Portal. To launch PSA services, you have to open the Submit Window in the Workflow Execution Portal and then select 'Service Workflows Only' option to filter PSA services from the list of workflows. After selecting a PSA service to submit, you can provide variable values that help configure this service then press submit.

**Review by mboussaa — CHANGES_REQUESTED** [review]
some changes

File: src/docs/PSA/references/Service_Execution.adoc [hunk 2]
```diff
@@ -2,5 +2,5 @@ PSA services can be instantiated in two different ways:
 
 1- Via the <<_psa_portal,ProActive Service Automation>> (PSA) Portal.
 
-2- Via standard workflows (also called *Management Workflows*). ProActive includes predefined standard workflows that manage PSA service lifecycles (e.g., launch and stop PSA services).
-These workflows are provided in the bucket *Service_Automation* under the project *Service Automation - Management*. Their names are constructed using the pattern *PSA_Service_Name* followed by the suffix *_Service*.
\ No newline at end of file
+2- Via Workflow Execution Portal. To launch PSA services, you have to open the Submit Window in the Workflow Execution Portal and then select 'Service Workflows Only'
```
Comment (mboussaa) [inline src/docs/PSA/references/Service_Execution.adoc:5]: remove line break after Service Workflows Only

**Review by mboussaa — APPROVED** [review]

**Review by aminelouati — APPROVED** [review]

**Comment by mboussaa** [comment]
restest this!

**Pull request merged**